FreeRTOS/FreeRTOS
# Migrated RL78/IAR port to EWRL78v3+

# RL78/IAR port update to EWRL78v3+

Description
-----------
The RL78/IAR port was based on earlier technology based on the UBROF object format. More recent versions of the IAR Embedded Workbench for Renesas RL78 use the industry standard ELF, which had impact on the interrupt vectors as well as for section placements.

Test Steps
-----------
Described in the new `Demo/RL78_multiple_IAR/README.md`

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
